src/odemis/acq/align/autofocus.py

```diff
@@ -713,19 +703,51 @@ def GetSpectrometerFocusingDetectors(focuser):
     return dets
 
 
-def _getSpectrometerFocusingComponents(focuser):
+def _findSparc2Focuser(align_mode: str) -> model.Actuator:
+    """
+    Find the correct focus actuator for the given alignment mode, based on the microscope model.
+    :param align_mode: see Sparc2AutoFocus
```

Comment:
If I check `Sparc2AutoFocus` I see the following options for align_mode: `OPM mode, spec-focus or spec-fiber-focus, streak-focus`. However here you also check for a few other modes. Should this docstring be extended, or the docstring for `Sparc2AutoFocus`, or something else?

Reply:
Additionally the docstring for `Sparc2AutoFocus` does not explain what align_mode is, could you update that?